src/unittest/python/tracker_tests.py

```diff
@@ -0,0 +1,73 @@
+from unittest import TestCase, mock
+
+from vyu import tracker
+
+
+class TestTrackerWaitForFixation(TestCase):
+
+    def setUp(self):
+        self.reader_patch = mock.patch('vyu.tracker.imageio.get_reader')
+        self.timer_patch = mock.patch('vyu.tracker.Timer')
+        self.i2pos_patch = mock.patch('vyu.tracker.image2position')
+        self.frames = [mock.Mock(), mock.Mock()]
+
+        self.mock_get_reader = self.reader_patch.start()
+        self.mock_timer = self.timer_patch.start()
+        self.mock_i2pos = self.i2pos_patch.start()
+
+        self.mock_get_reader.return_value.__iter__.return_value = self.frames
+        self.mock_timer_instance = self.mock_timer.return_value
+
+        self.mock_area = mock.MagicMock()
+
+    def tearDown(self):
+        mock.patch.stopall()
+
+    def test_init_creates_reader(self):
+        tracker.EyeTracker('ANY_CAMERA')
+        self.mock_get_reader.assert_called_once_with('ANY_CAMERA')
+
+    def test_time_is_up(self):
+        patience = 0.2
+
+        T = tracker.EyeTracker('ANY_CAMERA')
+        T.image2screen = mock.Mock()
+
+        self.mock_area.__contains__.return_value = True
+        self.mock_timer_instance.start_or_check.return_value = True
+
+        T.wait_for_fixation(self.mock_area, patience)
+
+        self.mock_i2pos.assert_called_once_with(self.frames[0])
+        T.image2screen.assert_called_once_with(self.mock_i2pos.return_value)
+        self.mock_area.__contains__.assert_called_once_with(
+            T.image2screen.return_value)
```

Comment:
Just to make sure this assert is making sure that the,
```python
if centroid in target_area:
```
in `line 20` of `tracker.py` is being called?

Reply:
Yes.